tracing/src/macros.rs

```diff
@@ -1840,11 +1840,15 @@ macro_rules! callsite {
 }
 
 #[macro_export]
-// TODO: determine if this ought to be public API?
+// TODO: determine if this ought to be public API?`
+// TODO(eliza): is the `has_been_set` check still necessary? the max
+// level check should be equivalent...
```

Comment:
I should benchmark this --- it's possible that the `has_been_set` check is still a faster path, but it's also more code...